sn_cli/src/subcommands/files/mod.rs

```diff
@@ -291,14 +291,21 @@ async fn upload_files(
             // log uploaded file information
             println!("**************************************");
             println!("*          Uploaded Files            *");
+            if !make_data_public {
```

Comment:
shall this alert msg to be put in the other code path as well ?

Reply:
The other codepath already got this alter msg, I copied the msg from there 👍